setup.py

```diff
@@ -3,11 +3,11 @@
 
 
 setup(
-    name='tungsten-ci-dashboard',
+    name='tungsten-ci-acid',
```

Comment:
Rollback this change. It's out of scope and in collision with https://github.com/codilime/acid/pull/1